pages/dkp/kommander/2.2/clusters/attach-cluster/requirements-for-attaching/index.md

```diff
@@ -72,6 +72,12 @@ To attach an existing EKS cluster, refer to the specific information in [Attach
 Consider the additional resource requirements for running the platform services you want Kommander to manage, and ensure that your existing clusters comply.
 -->
 
+## Attach clusters with an existing cert-manager installation
+
+If you are attaching clusters that already have cert-manager installed, the cert-manager `HelmRelease` provided by Kommander will fail to deploy, due to the existing cert-manager installation. As long as the pre-existing cert-manager functions as expected, you can ignore this failure. It will have no impact on the operation of the cluster.
+
+<p class="message--note"><strong>NOTE: </strong>Clusters provisioned using <code>dkp</code> (Konvoy2 clusters) have cert-manager installed. Kommander's cert-manager <code>HelmRelease</code> will fail to deploy on a Konvoy cluster.</p>
```

Comment:
Even EKS and AKS clusters are provisioned using DKP. Just say:

DKP Konvoy clusters have cert-manager already installed, hence Kommander's cert-manager will fail..

Otherwise looks good to me. @cwyl02

Reply:
done